client/.storybook/config.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { configure, addDecorator } from "@storybook/react";
```

Comment:
I don't think that `configure` is still needed. Can you try to remove it and see, if eslint throws errors? If not, you can sefely remove it.